test/dummy/db/schema.rb

```diff
@@ -19,6 +19,7 @@
     t.text "user_agent", null: false
     t.string "remote_addr", null: false
     t.string "token", null: false
+    t.boolean "claimed", default: false, null: false
```

Comment:
For bools like this I like to save them as timestamps. This provides a bit more information for _(as good as)_ free.

Let's change it to `claimed_at:datetime`, default `nil` and provide method in the `Session` class:

```ruby
def claimed?
  !!claimed_at
end
```